vnext/local-cli/generator-windows/templates/cpp/src/App.cpp

```diff
@@ -38,3 +42,39 @@ App::App() noexcept
 
     InitializeComponent();
 }
+
+/// <summary>
+/// Invoked when the application is launched normally by the end user.  Other entry points
+/// will be used such as when the application is launched to open a specific file.
+/// </summary>
+/// <param name="e">Details about the launch request and process.</param>
+void App::OnLaunched(LaunchActivatedEventArgs const& e)
+{
+    using super = AppT<App>;
```

Comment:
>using super [](start = 4, length = 11)

It would be better to define in .h file in case if developers want to implement more interface.